hosts/router/pppoe.nix

```diff
@@ -0,0 +1,40 @@
+{ config , ... }:
+
+{
+  # todo: find a way to load pppoe kernel module
+  
+  secrets.pppoe_pap_secrets.owner = "systemd-network"; # todo: find the right owner
+
+  environment.etc = {
+    "ppp/pap-secrets" = { 
+        source = config.secrets.pppoe_pap_secrets.path;
+        mode = "0400";
```

Comment:
Setting the mode is not necessary, because setting the source only creates a symlink, and the path it points to is already 0400.